src/test/resources/pages/instructorFeedbackEditEmpty.html

```diff
@@ -91,12 +91,12 @@
       <div class="row">
         <div class="col-sm-12">
           <span class="pull-right">
-            <a class="btn btn-primary btn-sm enable-edit-fs" data-original-title="Edit feedback session details" data-placement="top" data-toggle="tooltip" id="fsEditLink" title="">
+            <a class="btn btn-primary btn-sm enable-edit-fs" data-original-title="Edit feedback session details" data-placement="top" data-toggle="tooltip" id="fsEditLink" style="display: none;" title="">
```

Comment:
Example of a data attribute in the DOM: `data-original-title="Edit feedback session details"`. You'll need to find the JSP that generates this page and edit it to insert such an attribute (I suggest in the `<form>` element).